src/lib/openables.ts

```diff
@@ -246,7 +248,20 @@ const osjsOpenables: UnifiedOpenable[] = [
 		id: 22_993,
 		openedItem: getOSItem(22_993),
 		aliases: ['seed pack'],
-		output: Openables.SeedPack.table,
+		output: async (
+			args: OpenArgs
+		): Promise<{
+			bank: Bank;
+			message?: string;
+		}> => {
+			const { plantTier } = args.user.settings.get(UserSettings.Minion.FarmingContract) ?? defaultFarmingContract;
+			const realQty = args.quantity ?? 1;
```

Comment:
quantity cant be undefined, its always provided. its not a user-provided variable, its validated/checked.

Reply:
ah thanks for updating that for me :)